contrib/format-xml/src/main/java/org/apache/drill/exec/store/xml/XMLBatchReader.java

```diff
@@ -89,9 +89,9 @@ private void openFile(FileScanFramework.FileSchemaNegotiator negotiator) {
       reader = new XMLReader(fsStream, dataLevel, maxRecords);
       reader.open(rootRowWriter, errorContext);
     } catch (Exception e) {
-      throw UserException
+      throw UserException //lgtm[java/unused-format-argument]
```

Comment:
If you've fixed the cause of the lgtm warning, do we still need the comment?

Reply:
yep, my bad, forgot to remove it.